x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -77,6 +78,23 @@ const executeTimeoutValidator = (argData: ParsedArgData): true | string => {
   }
 };
 
+const filePathValidator = (argData: ParsedArgData): true | string => {
+  const emptyResult = emptyArgumentValidator(argData);
+  if (emptyResult !== true) {
+    return emptyResult;
+  } else if (
+    String(argData) &&
+    String(argData).trim().length > 0 &&
+    /\/|\\/.test(String(argData))
+  ) {
+    return true;
+  } else {
+    return i18n.translate('xpack.securitySolution.endpointConsoleCommands.invalidFilePath', {
+      defaultMessage: 'Argument must be a non-empty string representing a file path',
```

Comment:
maybe change it to `... representing a valid system path to a file or directory`?

Reply:
removed the path validation for now 99e13666cb030e03427e48b3bf935b09b293207d